test.js

```diff
@@ -1,3 +1,5 @@
+/* globals test, expect, fail */
```

Comment:
Please remove this comment. There are changes to made here, but we use ESLint. Please see #57.

Reply:
btw I don't expect you to make these changes! Just letting you know that I'm aware of the issue. Editing that file is pretty terrible in my IDE is pretty terrible right now. If I had time I'd migrate from jest to tape.